.github/workflows/pull-request-automation.yml

```diff
@@ -0,0 +1,14 @@
+on: pull_request
+name: Pull request automation
```

Comment:
Wrong name!

Reply:
Also looks like the file name might need to be updated.

Reply:
The _workflow_ has `on: pull_request` and so is named "Pull request automation". It uses an action called "Project management automation"

Reply:
ok ✅